### ISSUE TITLE
Color and Depth are not consistent?  

### ISSUE THREAD

**Issue by wangsen1312** [facebookresearch/pytorch3d#100]
It seems the color and depth are not consistent?
Y-axis Flipped?

![color_cow](https://user-images.githubusercontent.com/11545246/75933149-7442c800-5e36-11ea-9179-35b14633de0e.png)
![depth_cow](https://user-images.githubusercontent.com/11545246/75933152-76a52200-5e36-11ea-9092-445bbe751142.png)

my simple mehs render code is attached:

```
class MeshRendererWithDepth(nn.Module):
    def __init__(self, rasterizer, shader):
        super().__init__()
        self.rasterizer = rasterizer
        self.shader = shader
    def forward(self, meshes_world, **kwargs) -> torch.Tensor:
        fragments = self.rasterizer(meshes_world, **kwargs)
        images = self.shader(fragments, meshes_world, **kwargs)
        return images, fragments.zbuf
```

**Comment by nikhilaravi** [comment]
@wangsen1312 thanks for pointing this out. See #78 where we have addressed this issue. We are working on a fix which should be available soon.

**Comment by wangsen1312** [comment]
@nikhilaravi Got it.
